scripts/gen_manifests.sh

```diff
@@ -2,6 +2,11 @@
 
 WORKFLOW_ID=$1
 
+ENABLE_PERSISTENCE=false
```

Comment:
why `false` by default? Would not it make sense to have by default the same behaviour as the CI?

Reply:
This is useful for running the makefile manually.
we don't want to include it by default since the sonataflow-patch.yaml includes the persistence section by the pipeline and the `make push-manifsets`.
therefore if I'd like to build a workflow and test it in prod-mode locally, this PR become pretty useful.